components/DetailPageSidebar/DetailPageSidebar.js

```diff
@@ -0,0 +1,197 @@
+                        <p className="tag-description stage-tag-description" dangerouslySetInnerHTML={{ __html: i18n.deprecationWarning }}/>
+                    )}
+                    { artifact.tag === 'ibm_experimental' && (
+                        <div className="tag-container">
+                            { isStandardAccount && artifact.accountTag &&
+                                <div className="account-tag">{artifact.accountTagName}</div>
+                            }
+                            <div className="provider-tag">{artifact.tagName}</div>
+                            { artifact.stageTag &&
+                                <div className="stage-tag">{artifact.stageTagName}</div>
+                            }
+                        </div>
+                    )}
+                    { artifact.tag === 'ibm_experimental' && (
+                        <p className="tag-description" dangerouslySetInnerHTML={{ __html: i18n.experimentalDesc }}/>
+                    )}
+
+                    <p className="details-description">{artifact.description}</p>
+
+                    { artifact.tag !== 'ibm_experimental' &&
+                        <div className="artifact-margin-addition"></div>
+                    }
+                    { artifact.tag !== 'ibm_experimental' &&
+                        <div className="tag-container">
+                            { isStandardAccount && artifact.accountTag &&
+                                <div className="account-tag">{artifact.accountTagName}</div>
+                            }
+                            <div className="provider-tag">{artifact.tagName}</div>
+                            { artifact.availabilityTag && artifact.availabilityTag !== 'public' &&
+                                <div className="availability-tag">{artifact.availabilityTagName}</div>
+                            }
+                            { artifact.stageTag &&
+                                <div className="stage-tag">{artifact.stageTagName}</div>
+                            }
+                        </div>
+                    }
+                    {children}
+                    <div className="description-tag-container">
+                        { artifact.docURL &&
+                            <a className="description-tag-docs-link" href={artifact.docURL} target="_new">
+                                {i18n.viewDocs}
+                            </a>
+                        }
+                        { artifact.type === 'softlayer' && artifact.termsUrl &&
+                            <a className="description-tag-terms-link" href={artifact.termsUrl} target="_new">
+                                {i18n.viewTerms}
+                            </a>
+                        }
+                    </div>
+                    <div className="details-container">
+                        { artifact.author &&
```

Comment:
I would group this kind of `container` div creations into a function you can reuse - looks like each of these is the exact same syntax, barring a variable change.